tests/files/rbdigital/response_catalog_delta.json

```diff
@@ -42,7 +41,7 @@
         "audience": "Childrens",
         "genres": "JUVENILE NONFICTION / Biography & Autobiography / Sports & Recreation",
         "primaryGenre": "biography-autobiography-memoir,history,juvenile-nonfiction,sports",
-        "isbn": "9781615730186",
+        "isbn": "9780590543439",
```

Comment:
This was the same ISBN as "Tricks", which made it impossible to separate what happened to this book from what happened to "Tricks".